### DISCUSSION TITLE
Can Spelling Check Recognize Words in Multiple Languages?

### DISCUSSION THREAD

**Discussion by mrjasonm** [coteditor/CotEditor#1597] [Q&A]
I frequently use the app as a scratchpad for both words in Spanish and English. When my macOS system settings are set to EN-US, and I type a word in Spanish, it tries to autocorrect it to the closest English word, and vice-versa.
<img width="377" alt="Screenshot 2024-03-07 at 12 26 41 PM" src="https://github.com/coteditor/CotEditor/assets/62827390/f0d0df77-f15b-42d7-8f4c-4798cf0e7cf2">

Is there a way to incorporate spelling recognition for multiple languages? I envision that this could possibly be implemented as a setting, where users could click to check which languages to use for spelling/grammar correction, with a default to the current system setting?

I realize there could be platform (OS) limitations that could prevent this, but wanted to ask as it makes multi-language tasks incredibly frustrating.

**Comment by 1024jp** [comment]
CotEditor's spell checker uses the spell checker provided by macOS. Thus, by selecting Spelling and Grammar > Show Spelling and Grammar command in the Edit menu, you can access the GUI window for the spell check feature. In this panel, you can select the (natural) language for the spell check.

<img width="477" alt="Screenshot 2024-06-04 at 14 28 58" src="https://github.com/coteditor/CotEditor/assets/1165044/a6ebd1cd-c386-4b08-96a5-eff4445deb94">
